src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -35,47 +35,49 @@ def __call__(
         self.vae.to(torch_device)
         self.text_encoder.to(torch_device)
 
+        # get prompt text embeddings
+        text_input = self.tokenizer(prompt, padding="max_length", max_length=77, return_tensors="pt")
+        text_embeddings = self.text_encoder(text_input.input_ids.to(torch_device))[0]
+
+        do_classifier_free_guidance = guidance_scale > 1.0
```

Comment:
We are now ignoring values of `guidance_scale` < 1, I suppose that makes sense.

Reply:
If I understand correctly in the original paper the guidance scale `w` (in the paper called `"guidance strength"`) was chosen between [0, inf) - see: equation 6 here: https://arxiv.org/pdf/2207.12598.pdf) whereas it seems like CompVis used an adapted formula where `scale = 1 + w` so that scale = 1 => w = 0 => no classifier free guidance. 

In the Imagen paper they also use the CompVis notation where `w = 1 => no classifier free guidance`. Think we should be quite specific here what exactly we mean by `guidance_scale` then to avoid confusion.

IMO we should either 

```suggestion
         # here `guidance_scale` is defined analog to the guidance weight `w` of equation (2)
         # of the Imagen paper: https://arxiv.org/pdf/2205.11487.pdf . `guidance_scale = 1`
         # corresponds to doing no classifier free guidance.
        do_classifier_free_guidance = guidance_scale > 1.0
```

Reply:
In the CompVis paper they aren't very clear about it (no formula is shown: https://arxiv.org/pdf/2112.10752.pdf)

Reply:
Thanks a lot for tracking this down, @patrickvonplaten! I think the comment is very helpful and prevents uncertainty.